### PR TITLE
Labeling suite changes

### PR DESCRIPTION
**Pull request by JWittmeyer**
still open: Changes for model callback/external model naming

### CODE REVIEW

File: src/app/base/components/dropdown/dropdown-helper.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ import { FormArray } from "@angular/forms";
  * @keepDropdownOpen {boolean, optional} - stops the event propagation of the click event and therfore keeps the menu open
  * @buttonTooltip {string, optional} - adds a tooltip if defined
  * @buttonTooltipPosition {string, optional} - if empty defaults to right otherwise tooltip + position (e.g. tooltip-left)
+ * @buttonWhitespace {string, optional} - holds style parameter for whitespace handling (e.g. nowrap)
  * @isDisabled {boolean, optional} - disables the dropdown
```
Comment (lumburovskalina) [inline src/app/base/components/dropdown/dropdown-helper.ts:13]: buttonWhitespace is missing in the type DropdownOptions

Comment (JWittmeyer) [reply src/app/base/components/dropdown/dropdown-helper.ts:13]: added

**Comment by lumburovskalina** [comment]
When clicking on the 'related' link and hovering the label on the table a wrong row in the table is selected. Ex. on the picture 'yes' is selected but the table hovers 'Weak supervision' and in the table header 'no' label is selected.
- [ ] resolved

Screenshot 2023-02-06 at 17 53 47


_________________
couldn't reproduce

**Comment by lumburovskalina** [comment]
Changing the hover background of the table does not change the hover color of the table, it is the default green no matter which option is chosen. (after refreshing the page, the color is correct)
- [x] resolved

**Comment by lumburovskalina** [comment]
Tooltip for gold star is cut
- [x] resolved

Screenshot 2023-02-06 at 18 14 34

**Comment by lumburovskalina** [comment]
Hovering types (Manual, Weak Supervision, Heuristics..) gives an error in the console
- [x] resolved

Screenshot 2023-02-06 at 18 22 58

**Review by lumburovskalina — APPROVED** [review]

**Pull request merged**